EIPS/eip-5189.md

```diff
@@ -0,0 +1,248 @@
+---
+eip: <to be defined>
+title: Account Abstraction via Endorsed Operations
+description: An account abstraction proposal that avoids consensus-layer protocol changes while maintaining compatibility with existing smart contract wallets.
+author: Agustín Aguilar (@agusx1211), Philippe Castonguay (@phabc).
+discussions-to: https://ethereum-magicians.org/t/erc-account-abstraction-via-endorsed-operations/9799
+type: Standards Track
+category: ERC
+status: Draft
+created: 2022-06-29
+---
+
+## Abstract
+
+An account abstraction proposal that ensures compatibility with existing smart contract wallets and provides flexibility for alternative designs, while avoiding introducing changes to the consensus layer.
+
+Instead of defining a strict structure for meta-transactions, this proposal introduces the figure of `endorser` contracts. These smart contract instances are tasked with determining the quality of the submitted meta-transactions, thus safely helping bundlers determine if a meta-transaction should be kept in the mempool or not.
+
+Developers that intend to make their smart contract wallet compatible with this EIP must create and deploy an instance of an `endorser`; this instance must be seeded with a small amount of ETH to be burnt that incentivizes its good behavior.
```

Comment:
```suggestion
This EIP proposes a form of account abstraction that ensures compatibility with existing smart contract wallets and provides flexibility for alternative designs while avoiding introducing changes to the consensus layer. Instead of defining a strict structure for meta-transactions, this proposal introduces the figure of `endorser` contracts. These smart contract instances are tasked with determining the quality of the submitted meta-transactions, thus safely helping bundlers determine if a meta-transaction should be kept in the mempool or not. Developers that intend to make their smart contract wallet compatible with this EIP must create and deploy an instance of an `endorser`; this instance must be seeded with a small amount of ETH to be burnt that incentivizes its good behavior.
```

Make into a paragraph.